Dockerfile

```diff
@@ -6,12 +6,12 @@ COPY src ./src
 RUN mvn versions:set -DnewVersion=${version} && \
   mvn package -P assembly
 
-FROM adoptopenjdk/openjdk13:alpine-jre
+FROM adoptopenjdk/openjdk14:alpine-jre
+ENV JAVA_OPTS=""
 LABEL description="Artipie binary repository managment tool"
 LABEL maintainer="titantins@gmail.com"
 LABEL maintainer="g4s8.public@gmail.com"
 COPY --from=build_jar /jar/target/artipie-jar-with-dependencies.jar /usr/lib/artipie.jar
 COPY _config.yml /etc/artipie.yml
 EXPOSE 80
-ENTRYPOINT ["java", "-jar", "/usr/lib/artipie.jar"]
-CMD ["--config-file=/etc/artipie.yml", "--port=80"]
+CMD java $JVM_OPTS --enable-preview -XX:+ShowCodeDetailsInExceptionMessages -jar /usr/lib/artipie.jar --config-file=/etc/artipie.yml --port=80
```

Comment:
I use shell form because there is no different way to pass JVM_OPTS.

Reply:
@Sammers21 I know that. But the problem is still not fixed: you've introduced env-variable `JAVA_OPTS`, but uses `JVM_OPTS` (different name)

Reply:
@g4s8, just fixed in https://github.com/artipie/artipie/pull/323/commits/8ee88d3c1fd645e71abd10bc957c6f98ad655f98